travel-agency-persistence/src/test/java/cz/fi/muni/pa165/travelagency/dao/ReservationDaoTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.fi.muni.pa165.travelagency.dao;
+
+import cz.fi.muni.pa165.travelagency.EmbeddedDatabaseSpringConfig;
+import cz.fi.muni.pa165.travelagency.entity.Customer;
+import cz.fi.muni.pa165.travelagency.entity.Reservation;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+import org.springframework.test.context.support.DirtiesContextTestExecutionListener;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+import org.springframework.transaction.annotation.Transactional;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author Diana Vilkolakova
+ */
+@ContextConfiguration(classes = EmbeddedDatabaseSpringConfig.class)
```

Comment:
Tu bude @ContextConfiguration(locations = "/SpringXMLConfig.xml")
Aby ti to potom fungovalo, treba, aby si si updatla SpringXMLConfig.xml